src/modules/jobs/dto/createJobDto.ts

```diff
@@ -0,0 +1,84 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsEnum, IsNotEmpty, IsString } from 'class-validator';
+
+// Will be added after merging Freelancer and Employer entity
+
+// import { Category } from 'src/database/entities/category.entity';
+// import { Country } from 'src/database/entities/country.entity';
+// import { Skill } from 'src/database/entities/skill.entity';
+import {
+  AvailableAmountOfHours,
+  EmploymentType,
+  HourRate,
+  LevelEnglish,
+  WorkExperience,
+} from 'src/database/enums';
+
+export class CreateJobDto {
+  @ApiProperty({ example: 'Java script developer for a big  educational project' })
+  @IsString()
+  @IsNotEmpty()
+  title: string;
+
+  @ApiProperty({ example: 'Any text wich discribes current job' })
+  @IsString()
+  @IsNotEmpty()
+  description: string;
+
+  // Will be added after merging Freelancer and Employer entity
+
+  // @ApiProperty()
+  // @IsNotEmpty()
+  // @IsArray()
+  // @ArrayMinSize(5)
+  // countries: Country[];
+
+  // Will be added after merging Freelancer and Employer entity
+
+  // @ApiProperty()
+  // @IsNotEmpty()
+  // category: Category;
+
+  @ApiProperty({ example: 'Fullstack developer' })
+  @IsNotEmpty()
+  @IsString()
+  position: string;
+
+  @ApiProperty({ example: 'remote' })
+  @IsNotEmpty()
+  @IsEnum(EmploymentType)
+  employmentType: EmploymentType;
+
+  @ApiProperty({ example: 'less than 500$' })
+  @IsNotEmpty()
+  @IsEnum(HourRate)
+  hourRate: HourRate;
+
+  @ApiProperty({ example: 'part time' })
+  @IsNotEmpty()
+  @IsEnum(AvailableAmountOfHours)
+  availableAmountOfHours: AvailableAmountOfHours;
+
+  @ApiProperty({ example: 'less than 1 year' })
+  @IsNotEmpty()
+  @IsEnum(WorkExperience)
+  workExperience: WorkExperience;
+
+  // Will be added after merging Freelancer and Employer entity
+
+  // @ApiProperty()
+  // @IsNotEmpty()
+  // @IsArray()
+  // @ArrayMinSize(3)
+  // skills: Skill[];
```

Comment:
it should not be in PR, or dont add it or add it, why add and comment. this time I will pass it but next I will not

Reply:
I understand. I should have left a comment for the developers, as I added in other files, but I missed it.